src/pages/blog/2018-11-18-spustili-sme-eshop-na-openbazaare.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Spustili sme eshop na Openbazaare'
+cover: /assets/open-bazaar.jpg
+author: Michal Takac
+tags:
+  - oznamy, tutorialy
```

Comment:
Novy tag sa vytvory cez novy odrazku, + by som to mozno nazval slovensky - navody:

```
tags:
  - oznamy
  - navody
```